src/app/modules/data-selection/constraint-components/constraint/constraint.component.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  <div *ngIf="constraint.getConstraintType() == 'combination-constraint'">
```

Comment:
Couldn't this be done with instanceof? Also, it may be nicer if we could instantiate the constraint components on the fly, instead of hard-coding them in a switch-type check.